src/js/roleToAccountMapping.js

```diff
@@ -23,6 +23,19 @@ var arrDeletedConflictCount = [];
 var etagValue  = '';
 var isDeleted = false;
 
+roleToAccountMapping.prototype.setRoleName = function(name) {
+	roleName = name;
```

Comment:
These global variables (roleName, boxName) are used within this files, too. Should replace them with set/get methods.